doc/index.rst

```diff
@@ -151,3 +151,23 @@ This bundle already installed for you the right binary. However, if you already
 
     symfonycasts_sass:
         binary: 'node_modules/.bin/sass'
+
+Register additional load paths
```

Comment:
This section of the doc should probably have a link to https://sass-lang.com/documentation/at-rules/use/#finding-the-module

Reply:
Added a link in the documentation.